setup-jest.js

```diff
@@ -0,0 +1 @@
+jest.retryTimes(2);
```

Comment:
Each batch of tests is executed by a worker pool containing 3 workers, so if a test fails we want to be able to re-run that tests without having to wait for all of the other tests to finish up. 
This will allow a failed test to run on a worker that frees up.